main-app/app/scripts/modules.js

```diff
@@ -0,0 +1,33 @@
+angular.module('Core.MyModule', []);
+angular.module('Server.MyModule',['Core.MyModule']);
+angular.module('Tombola.MyModule', ['Server.MyModule', 'ui.router', 'Core.MyModule'])
+    .config(function($stateProvider, $urlRouterProvider) {
+
+        $urlRouterProvider.otherwise("/PlayerSelect");
+
+        $stateProvider
+            .state('route1', {
```

Comment:
Next cut - rename the states to something meaningful - don't want to have to remember route4 id the winner's state etc...